### PR TITLE
Some more minor code inspection fixes

### PR DESCRIPTION
**Pull request by fviernau**

### CODE REVIEW

File: docs/config-file-copyright-garbage-yml.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ You can use the [copyright-garbage.yml example](../examples/copyright-garbage.ym
 
 ### When to Use
 
-No scanner is perfect and many detect code statements or binary code as copyright holders,
+No scanner is perfect, and many detect code statements or binary code as copyright holders,
```
Comment (sschuberth) [inline docs/config-file-copyright-garbage-yml.md:9]: Actually, I believe we must not use a comma after "and" here as the second clause is not independent from the first one, see the (incorrect) "Sam tossed the ball, and watched the dog chase it." example [here](https://www.grammarly.com/blog/comma-before-and/).

File: docs/config-file-resolutions-yml.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ by providing a reason why they are acceptable and can be ignored.
 You can use the [resolutions.yml example](../examples/resolutions.yml) as the base configuration file for your scans.
 
 ### When to Use Resolutions
-Resolutions should be used when it is impossible to solve an issue or a fix is planned for a later time.
+Resolutions should be used when it is impossible to solve an issue, or when a fix is planned for a later time.
```
Comment (sschuberth) [inline docs/config-file-resolutions-yml.md:9]: For a similar reason as stated above, I believe we must not use a comma here.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: docs/config-file-copyright-garbage-yml.md (hunk 1 above)
Comment (fviernau) [reply docs/config-file-copyright-garbage-yml.md]: Right. Done.

File: docs/config-file-resolutions-yml.md (hunk 2 above)
Comment (fviernau) [reply docs/config-file-resolutions-yml.md]: Right. Done.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**